### PR TITLE
chore(Dagre Graph): Added Basic Workflow Preview and install-experiment steps for Predefined Workflows 🖼️

### PR DESCRIPTION
**Pull request by S-ayanide**
## Proposed changes

* Modified Extract Steps utility to append install experiment and revert chaos for predefined experiments
* Added Basic Workflow Preview
* Fixed Stepper active and completed icon color

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx [hunk 1]
```diff
@@ -1,22 +1,141 @@
...
+import { createLabel } from './createLabel';
+import useStyles from './styles';
+
+interface GraphData {
+  nodes: d3Node[];
+  links: d3Link[];
+}
 
 interface WorkflowPreviewProps {
   isCustom: boolean;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx:16]: can isCustom be changed to a more descriptive variable

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx (hunk 1 above)
Comment (S-ayanide) [reply litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx:16]: What name would you suggest? 
This variable only denotes if the Dagre graph is for a custom workflow or not.

Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx:16]: how about `isCustomWorkflow` or `isUserDefinedWorkflow` ?

Comment (S-ayanide) [reply litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx]: Updated the variable

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**